upgrade/sql/1.7.8.0.sql

```diff
@@ -199,3 +199,11 @@ CREATE TABLE IF NOT EXISTS `PREFIX_feature_flag` (
 INSERT INTO `PREFIX_feature_flag` (`name`, `state`, `label_wording`, `label_domain`, `description_wording`, `description_domain`)
 VALUES
 	('product_page_v2', 0, 'Experimental product page', 'Admin.Advparameters.Feature', 'This page benefits from increased performance and includes new features such as a new combination management system. Please note this is a work in progress and some features are not available yet.', 'Admin.Advparameters.Help');
+
+DROP INDEX id_shop ON `PREFIX_shop_url`;
+DROP INDEX full_shop_url ON `PREFIX_shop_url`;
+DROP INDEX full_shop_url_ssl ON `PREFIX_shop_url`;
+ALTER TABLE `PREFIX_shop_url` CHANGE id_shop_url id_shop_url INT AUTO_INCREMENT NOT NULL, CHANGE id_shop id_shop INT NOT NULL;
+CREATE INDEX IDX_279F19DA274A50A0 ON `PREFIX_shop_url` (id_shop);
```

Comment:
Surprisingly, key names seem to be the same between two executions of the same command.

```
mysql> DESCRIBE ps_shop_url;
+--------------+--------------+------+-----+---------+----------------+
| Field        | Type         | Null | Key | Default | Extra          |
+--------------+--------------+------+-----+---------+----------------+
| id_shop_url  | int(11)      | NO   | PRI | NULL    | auto_increment |
| id_shop      | int(11)      | NO   | MUL | NULL    |                |
| domain       | varchar(150) | NO   |     | NULL    |                |
| domain_ssl   | varchar(150) | NO   |     | NULL    |                |
| physical_uri | varchar(64)  | NO   |     | NULL    |                |
| virtual_uri  | varchar(64)  | NO   |     | NULL    |                |
| main         | tinyint(1)   | NO   |     | NULL    |                |
| active       | tinyint(1)   | NO   |     | NULL    |                |
+--------------+--------------+------+-----+---------+----------------+
8 rows in set (0.00 sec)

mysql> SHOW INDEX FROM ps_shop_url;
+-------------+------------+----------------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| Table       | Non_unique | Key_name             | Seq_in_index | Column_name | Collation | Cardinality | Sub_part | Packed | Null | Index_type | Comment | Index_comment |
+-------------+------------+----------------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| ps_shop_url |          0 | PRIMARY              |            1 | id_shop_url | A         |           0 |     NULL | NULL   |      | BTREE      |         |               |
| ps_shop_url |          1 | IDX_279F19DA274A50A0 |            1 | id_shop     | A         |           0 |     NULL | NULL   |      | BTREE      |         |               |
+-------------+------------+----------------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
2 rows in set (0.00 sec)
```